modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/extension/dialog_definition.js

```diff
@@ -14,22 +14,34 @@
 
 CKEDITOR.on('dialogDefinition', (event) => {
 	if (event.editor === ckEditor) {
+		var boundingWindow = event.editor.window;
+
 		var dialogDefinition = event.data.definition;
 
 		var onShow = dialogDefinition.onShow;
 
-		dialogDefinition.onShow = function () {
+		var dialog;
+
+		dialogDefinition.onShow = function() {
+			dialog = this;
```

Comment:
Instead of doing:

```javascript
dialog = this;
```

on line 26, we can do  

```javascript
var dialog = event.data.dialog;
```

on line 20 ( just underneath `var dialogDefinition = event.data.definition`;)

This avoids creating an `instance` variable inside the `centerDialog` function and makes it a bit clearer that what we're calling the `move` function on a `dialog` "object" and not on `this` (which doesn't have the same value depending on where it's assigned)

Reply:
Thanks that's a lot nicer, I tried a few option but never checked the original event object.